app/views/docs/permissions.phtml

```diff
@@ -44,60 +43,62 @@
     </thead>
     <tbody>
     <tr>
-        <td>
-            role:all
-            <p class="margin-top-small text-fade">version >= 0.12</p>
-        </td>
-        <td>Wildcard permission. Gives anyone read or write access.</td>
+        <td>Role.any()</td>
+        <td>Grants access to anyone.</td>
+    </tr>
+    <tr>
+        <td>Role.guests()</td>
+        <td>Grants access to any guest user without a session. Authenticated users don't have access to this role.</td>
+    </tr>
+    <tr>
+        <td>Role.users()</td>
+        <td>Grants access to any authenticated or anonymous user.</td>
     </tr>
     <tr>
-        <td>user:[USER_ID]</td>
-        <td>Access to a specific user by his UID.</td>
+        <td>Role.user([USER_ID])</td>
+        <td>Grants access to a specific user by user ID.</td>
     </tr>
     <tr>
-        <td>team:[TEAM_ID]</td>
-        <td>Access to any member of the specific team. To gain access to this permission, the user must be the team creator (owner), or receive and accept an invitation to join this team.</td>
+        <td>Role.team([TEAM_ID])</td>
+        <td>Grants access to any member of the specific team. To gain access to this permission, the user must be the team creator (owner), or receive and accept an invitation to join this team.</td>
+    </tr>
+    <tr>
+        <td>Role.team([TEAM_ID], [ROLE])</td>
+        <td>Grants access to any member who possesses a specific role in a team. To gain access to this permission, the user must be a member of the specific team and have the given role assigned to them. Team roles can be assigned when inviting a user to become a team member.</td>
+    </tr>
+    </tbody>
+</table>
+
+<h2 id="permission-types"><a href="/docs/permissions#permission-types">Permission Types</a></h2>
+<p>In Client and Server SDKs, you will find a <b>Permission</b> class with helper methods for each role described below.</p>
+<table class="full text-size-small">
+    <thead>
+    <tr>
+        <td style="width: 180px">Type</td>
+        <td>Description</td>
     </tr>
+    </thead>
+    <tbody>
     <tr>
-        <td>team:[TEAM_ID]/[ROLE]</td>
-        <td>Access to any member who possesses a specific role in a team. To gain access to this permission, the user must be a member of the specific team and have the given role assigned to him or her. Team roles can be assigned when inviting a user to become a team member.</td>
+        <td>Permission.read()</td>
+        <td>Access to view a resource. Does not apply to functions.</td>
```

Comment:
> Does not apply to functions.

I think we can remove all instances of this as functions won't use permissions, only roles